_translator-files/po/sv-SE/Installation-for-Android.po

```diff
@@ -1,131 +1,132 @@
 # SOME DESCRIPTIVE TITLE
 # Copyright (C) YEAR Free Software Foundation, Inc.
 # This file is distributed under the same license as the PACKAGE package.
-# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
-#
-#, fuzzy
+# Daniel <daniel@mailsystem.net>, 2024.
+# ignotus <ignotus666@users.noreply.hosted.weblate.org>, 2024.
 msgid ""
 msgstr ""
 "Project-Id-Version: PACKAGE VERSION\n"
-"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
-"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"PO-Revision-Date: 2024-08-06 18:09+0000\n"
+"Last-Translator: Daniel <daniel@mailsystem.net>\n"
 "Language-Team: LANGUAGE <LL@li.org>\n"
-"Language: \n"
+"Language: sv-SE\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=n != 1;\n"
+"X-Generator: Weblate 5.7-dev\n"
 
 #. type: Yaml Front Matter Hash Value: lang
 #: ../wiki/en/Installation-for-Android.md:1
 #, no-wrap
 msgid "en"
-msgstr ""
+msgstr "sv-SE"
 
 #. type: Yaml Front Matter Hash Value: layout
 #: ../wiki/en/Installation-for-Android.md:1
 #, no-wrap
 msgid "wiki"
-msgstr ""
+msgstr "wiki"
 
 #. type: Yaml Front Matter Hash Value: permalink
 #: ../wiki/en/Installation-for-Android.md:1
 #, no-wrap
 msgid "/wiki/Installation-for-Android"
-msgstr ""
+msgstr "/wiki/Installation-for-Android"
 
 #. type: Title #
 #: ../wiki/en/Installation-for-Android.md:1
 #: ../wiki/en/Installation-for-Android.md:11
 #, markdown-text, no-wrap
 msgid "Installation for Android"
-msgstr ""
+msgstr "Installation för Android"
 
 #. type: Plain text
 #: ../wiki/en/Installation-for-Android.md:9
 #, markdown-text
 msgid "{% include breadcrumb.html root=\"Using Jamulus\" branch1=\"Getting Started\" branch1-url=\"Getting-Started\" %}"
-msgstr ""
+msgstr "{% include breadcrumb.html root=\"Använda Jamulus\" branch1=\"Komma igång\" branch1-url=\"Getting-Started\" %}"
 
 #. type: Plain text
 #: ../wiki/en/Installation-for-Android.md:14
 #, markdown-text
 msgid "Make sure you've already read the [Getting Started](Getting-Started) page."
-msgstr ""
+msgstr "Se till att du redan har läst sidan [Getting Started](Getting-Started)."
```

Comment:
Translate text in square brackets.

Reply:
Possibly "komma igång" as it's in the page title:
> Setup - getting started with Jamulus

becomes
> Installation - komma igång med Jamulus